docs/QUICKSTART.md

```diff
@@ -0,0 +1,115 @@
+# Quick Start Guide
+
+The quick start guide will help you use a local docker-compose file to start Locust, a mediator, and an Aca-Py instance. This guide will help you setup and configure the local environment and run Locust easily.
+
+While this is a quick start guide to familiarize yourself with Locust testing for Aries based protocols, this project is not designed to manage or maintain any Aries environment, but to apply a load to it. 
+
+## Setting up the environment
+
+Copy the sample.demo.env file to .env
+
+The start the demo docker-compose by running the following command.
+
+`docker-compose -f docker-compose.demo.yml up`
```

Comment:
I think the modern way to do this is `docker compose` — no dash.

Reply:
On Ubuntu 22.04.3 LTS it is still `docker-compose`. `docker compose` doesn't work for me.